securedrop/journalist_templates/col.html

```diff
@@ -1,4 +1,7 @@
 {% extends "base.html" %}
+
+{% block tab_title %}Source Submissions{% endblock %}
```

Comment:
This is a fine generalized `title`!  We could, however, use `source.journalist_designation` here, which would have the added benefit of making different sources' `col` listings navigable in Tor Browser history.  @zenmonkeykstop, do you have any strong feelings either way?

Reply:
SGTM, I'll update the commit if we get sign-off on using it.

Reply:
No issues with this that I can see.

Reply:
Added a third commit which implements this change

Reply:
Looks good!